ccache.c

```diff
@@ -1242,6 +1277,23 @@ to_cache(struct args *args)
 		}
 	}
 
+	if (generating_stackusage) {
+		// GCC won't generate notes if there is no code.
```

Comment:
I'm wondering if this applies to stackusage the way it did for coverage... (notes)